.github/workflows/4-deploy-to-prod.yml

```
@@ -0,0 +1,103 @@
+name: Deploy to prod
+on:
+  push:
+    branches:
+      - 'releases/prod'
+
+jobs:
+
+  tag:
+    runs-on: ubuntu-latest
+    outputs:
+      new_tag: ${{ steps.github-tag-action.outputs.new_tag }}
```

If we're aliasing it anyway, why not just call it `tag`?

```suggestion
      tag: ${{ steps.github-tag-action.outputs.new_tag }}
```